src/sentry/snuba/discover.py

```diff
@@ -1040,7 +942,7 @@ def histogram_query(
         elif array_column == "span_op_breakdowns":
             key_column = "array_join(span_op_breakdowns_key)"
             histogram_function = get_span_op_breakdown_name
-        else:
+        elif array_column is None:
```

Comment:
It shouldn't https://github.com/getsentry/sentry/pull/74106/files#diff-f9a8d9bbcb1bf33b928235196e41ca3666cc8c508bbb46d3238aae26d3e13a6cR938 only returns `"measurements", "span_op_breakdowns" or None`

Reply:
Isn't this strictly worse though? Because python doesn't force you to check all possible cases, it's entirely possible the function changes its return type (in this case, it's not likely to change), and then fails here.

So this specific use case is probably okay but is there a reason not to leave this as `else` as a catch-all?

Reply:
That's a good point, i'll switch this to Else